test/Pair.t.sol

```diff
@@ -417,14 +417,14 @@ contract PairTest is BaseTest {
 
         voter.vote(1, pools, weights);
         assertEq(voter.usedWeights(1), escrow.balanceOfNFT(1)); // within 1000
-        assertEq(bribe.balanceOf(1), uint256(voter.votes(1, address(pair))));
+        assertEq(xbribe.balanceOf(1), uint256(voter.votes(1, address(pair))));
```

Comment:
Question: Here we are checking that external bribe is receiving fees & bribes correctly?